charts/exivity/templates/migration/job.yaml

```diff
@@ -14,34 +18,6 @@ spec:
         app.kubernetes.io/component: migration
         {{- include "exivity.labels" $ | indent 8 }}
     spec:
-      initContainers:
-        - name: wait-for-server
-          image: {{ include "exivity.image" (set $ "name" "dbInit") }}
-          imagePullPolicy: {{ .Values.service.dbInit.pullPolicy | default .Values.service.pullPolicy | default "IfNotPresent" }}
-          command:
-            - "/wait.sh"
-          args:
-            - "--host"
-            - "$(PG_HOST)"
-            - "--port"
-            - "$(PG_PORT)"
-            - "--dbname"
-            - "$(PG_DATABASE)"
-            - "--username"
-            - "$(PG_USER)"
-          env:
-            - name: PG_HOST
-              value: {{ $.Values.postgresql.host | default (printf "%s-postgresql" (include "exivity.fullname" $ )) | quote }}
-            - name: PG_PORT
-              value: {{ $.Values.postgresql.port | default 5432 | quote}}
-            - name: PG_DATABASE
-              value: {{ $.Values.postgresql.global.postgresql.auth.database | quote }}
-            - name: PG_USER
-              value: {{ $.Values.postgresql.global.postgresql.auth.username | quote }}
-            - name: PG_SSLMODE
-              value: {{ $.Values.postgresql.sslmode | default "disable" | quote  }}
-          resources:
-            {{- toYaml .Values.service.dbInit.waitForServer.resources | nindent 12 }}
```

Comment:
Did you check that on a fresh install the DB is fully set up before it runs the hook, we had troubles with this in the past.

Reply:
I will have a proper check. But it should be alright because of the helm hook order. The Migration Job will be executed after the Exivity deployment and all external components are successfully deployed.